src/NuGet.Core/NuGet.Protocol/Converters/V3SearchResultsConverter.cs

```diff
@@ -0,0 +1,106 @@
+using Newtonsoft.Json;
+using NuGet.Protocol.Model;
+using System;
+
+namespace NuGet.Protocol.Converters
+{
+    public class V3SearchResultsConverter : JsonConverter
+    {
+        public int Take { get; set; }
```

Comment:
This isn't used anywhere outside the class, so it should be a private field

```suggestion
        private readonly int _take;
```

Reply:
Ok. Changed to private.